src/python/examples/CMakeLists.txt

```diff
@@ -29,6 +29,8 @@ cmake_minimum_required (VERSION 3.18)
 if(${TRITON_ENABLE_PYTHON_HTTP})
   install(
     PROGRAMS
+      simple_http_aio_infer_client.py
+      simple_http_aio_test_client.py
```

Comment:
Similar comment as on grpc side. Please move these tests to unitttest.

Reply:
Moved